src/main/resources/config.json

```diff
@@ -58,35 +48,50 @@
-		},
-		"akka": {
-			"akka.http.server.request-timeout": "45s",
-      "akka.http.server.server-header": "" // Removes the Server header from response.
-			// add additional entries like: "akka.http.server.max-connections": 512
-			// see: https://doc.akka.io/docs/akka-http/current/configuration.html
-			// and see: https://doc.akka.io/docs/akka/current/general/configuration-reference.html
-			// can use this to see all of the akka config: "akka.log-config-on-start": "on"
-		}
-	}
+    },
+    "defaults": {
+      "businessPolicy": {
+        "check_agreement_status": 1800,
+        "missing_heartbeat_interval": 1800  // used if the service.nodeHealth section is omitted
+      },
+      "msgs": {
+        "expired_msgs_removal_interval": 1800  // number of seconds between deletions of expired node and agbot messages
+      },
+      "pattern": {
+        "missing_heartbeat_interval": 1800,  // used if the services.nodeHealth section is omitted
+        "check_agreement_status": 1800
+      }
+    },
+    "limits": {
+      "maxAgbots": 1000,            // maximum number of agbots 1 user is allowed to create, 0 for unlimited
+      "maxAgreements": 0,           // maximum number of agreements 1 node or agbot is allowed to create, 0 for unlimited
+      "maxBusinessPolicies": 5000,  // maximum number of business policies 1 user is allowed to create, 0 for unlimited
+      "maxMessagesInMailbox": 0,    // maximum number of msgs currently in 1 node or agbot mailbox (the sending side is handled by rate limiting), 0 for unlimited
+      "maxNodes": 45000,            // maximum number of nodes 1 user is allowed to create, 0 for unlimited
+      "maxPatterns": 1000,          // maximum number of patterns 1 user is allowed to create, 0 for unlimited
+      "maxServices": 1000           // maximum number of services 1 user is allowed to create, 0 for unlimited
+    },
+    "logging": {
+      "level": "INFO"  // For possible values, see http://logback.qos.ch/apidocs/ch/qos/logback/classic/Level.html
+    },
+    "resourceChanges": {
+      "cleanupInterval": 3600,  // number of seconds between pruning the resourcechanges table in the db of expired changes - 3600 is 1 hour
+      "maxRecordsCap": 10000,   // maximum number of records the notification framework route will return
+      "ttl": 14400              // number of seconds to keep the history records of resource changes (14400 is 4 hours). When agents miss 1 or more heartbeats, they reset querying the /changes route, so they do not need very old entries.
+    },
+    "root": {
+      "enabled": true,  // if set to false it will not honor the root credentials
+      "password": ""    // set this in your own version of this config file
+    },
+    "service": {
+      "host": "0.0.0.0",
+      "portEncrypted": 8080,                   // Services HTTPS traffic
+      "portUnencrypted": null,                 // Services HTTP traffic
```

Comment:
Why is there not a default for this value? This doesn't seem backward compatible, unless i'm misunderstanding the changes. Since the current config value `service.port` is really the unencrypted port, seems like you should use that config value, instead of introducing a new parameter for that.

Also, will the exchange be able to listen on both the encrypted and unencrypted ports at the same time, or only 1 or the other? The golang mgmt hub components are the latter, so i'm fine if that will be the behavior of the exchange to, in fact it will make it simpler and more consistent.

The all-in-1 environment currently assumes the exchange binary is listening on port 8080 inside the container: https://github.com/open-horizon/devops/blob/0807ca00d90ecad1d39b5410756d45da7c795219/mgmt-hub/docker-compose.yml#L47 . I would like that to not break when this PR is merged/built.

Also, when i add support to all-in-1 to configure the mgmt hub components for TLS, i was planning to configure all of them to listen on the same port as they would have for non-TLS. (Thus 1 of my questions above.)